packages/renderer/package.json

```diff
@@ -3,26 +3,33 @@
   "version": "3.1.17",
   "license": "MIT",
   "description": "Create PDF files on the browser and server",
-  "types": "index.d.ts",
```

Comment:
No longer needed?

Reply:
Not only no longer needed, but actively harmful for type resolution in TypeScript. Instead, each bundled .js file gets its own .d.ts sibling file. This makes the bundle larger, unfortunately, but this is the reason we'll be seeing a vast improvement on arethetypeswrong after this PR.